test/notifications/open_test.dart

```diff
@@ -202,6 +206,22 @@ void main() {
         eg.dmMessage(from: eg.otherUser, to: [eg.selfUser]));
     }, variant: const TargetPlatformVariant({TargetPlatform.android, TargetPlatform.iOS}));
 
+    testWidgets('changes last visited account', (tester) async {
+      addTearDown(testBinding.reset);
+      await testBinding.globalStore.add(
+        eg.selfAccount, eg.initialSnapshot(realmUsers: [eg.selfUser]));
+      await testBinding.globalStore.add(
+        eg.otherAccount, eg.initialSnapshot(realmUsers: [eg.otherUser]),
+        markLastVisited: false);
+      await prepare(tester);
+      check(testBinding.globalStore).lastVisitedAccount
+        .isA<OptionSome<Account?>>().value.equals(eg.selfAccount);
```

Comment:
```suggestion
      check(testBinding.globalStore).lastVisitedAccount
        .isA<OptionSome<Account?>>().value.identicalTo(eg.selfAccount);
```

I think? From Greg's comment: https://github.com/zulip/zulip-flutter/pull/1784#discussion_r2292236441

(here and elsewhere)

Reply:
Oh, I see this doesn't work because `TestGlobalStore.add` makes a copy of the account its passed, and inserts that. Seems fine to stick with `equals` then.